e2e/tests/test_e2e.py

```diff
@@ -165,6 +166,7 @@ def test_overwrite_pooler_deployment(self):
             }
         })
 
+        time.sleep(10)
```

Comment:
no more sleeps, artificially slowing things down. lets talk about the issue here